tests/Unit/RmDirStepTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+use WordPress\Blueprints\Model\Builder\RmDirStepBuilder;
+use WordPress\Blueprints\Runner\Step\RmDirStepRunner;
+
+it('should remove a directory', function () {
+    $path = __DIR__ . "/dir";
```

Comment:
Let's give it a name that communicates the intention, say 
```suggestion
    $dirToRemove = __DIR__ . "/dir";
```

Reply:
Done. But with the resolving... I am not certain it looks good now. WBY?

Reply:
Renaming to agreed upon convention.